sdk/models.cc

```diff
@@ -0,0 +1,28 @@
+#include "models.h"
+
+std::ostream& iggy::models::operator<<(std::ostream& os, const iggy::models::Stats& stats) {
+    // Write the size of the string, then the string itself
+    size_t size = stats.os_version.size();
+    os.write(reinterpret_cast<const char*>(&size), sizeof(size));
+    os.write(stats.os_version.c_str(), size);
```

Comment:
keep in mind that iggy uses UTF8 strings - same as whole Rust 🦀 
that includes strings sent over the wire

Reply:
This is dummy code that should not have been part of the commit, I will take it out for now.

Reply:
Introduced icu into the build and put in basic checks here.